src/plugins/workspace/public/components/workspace_form/workspace_detail_form_details.tsx

```diff
@@ -129,6 +129,7 @@ export const WorkspaceDetailFormDetails = ({
           error={formErrors.color?.message}
         >
           <EuiCompressedColorPicker
+            mode="swatch"
             color={formData.color}
```

Comment:
This still allow user to type whatever color they want

Reply:
Maybe we could add a rule for color validation when submitting like this: `euiPaletteColorBlind().includes(color)`

Reply:
should we use same component as create page? for now they are different.

Reply:
I‘ve tried the validation and it works on both the create page and the detail page